packages/m365_defender/data_stream/event/fields/agent.yml

```
@@ -5,9 +5,12 @@
   footnote: 'Examples: If Metricbeat is running on an EC2 host and fetches data from its host, the cloud info contains the data about this machine. If Metricbeat runs on a remote machine outside the cloud and fetches data from a service running in the cloud, the field contains cloud data from the machine the service is running on.'
   type: group
   fields:
-    - name: image.id
+    - name: instance.id
       type: keyword
       description: Image ID for the cloud instance.
+    - name: image.id
+      type: keyword
+      description: Instance ID of the host machine.
```

```suggestion
    - name: image.id
      type: keyword
      description: Image ID for the cloud instance.
    - name: instance.id
      type: keyword
      description: Instance ID of the host machine.
```
(or something similar; this still doesn't look right, so please check what it is that you mean here)